### DISCUSSION TITLE
League of Garmin Promo Post

### DISCUSSION THREAD

**Discussion by gunjanmarwah** [philosowaffle/peloton-to-garmin#576] [General]
Hey there, I just got accepted to be part of the League of Garmin ambassador team and am thinking about doing a social post about this process and promoting it for other Garmin + Peloton users. Wanted to work with someone on talk tract and also make sure I tag any relevant instagram accounts for contributors. Please let me know @philosowaffle !

**Comment by philosowaffle** [comment]
Hello @gunjanmarwah! It's pretty much just me doing the majority of the contributions, but you can see an exhaustive list of everyone who has helped out over the years [here](https://github.com/philosowaffle/peloton-to-garmin/graphs/contributors).

Some special callouts from the contributors list:
1. @dkuwahara - [ Initially created](https://github.com/philosowaffle/peloton-to-garmin/commits?author=dkuwahara) the option to upload via [Github Action](https://philosowaffle.github.io/peloton-to-garmin/v4.0.0/install/github-action/).
2. @anlesk - Has added [notable enhancements](https://github.com/philosowaffle/peloton-to-garmin/commits?author=anlesk) to the GitHub action

I'm not very active on social media but my LinkedIn and Blog urls can both be found on my GitHub Profile.

Thank you for considering sharing this project with your audience!

**Comment by anlesk** [comment]
Hi @gunjanmarwah,
My contribution was not that notable (thanks for the mention though!), but the work @philosowaffle did to make it all happen is amazing.
I'd double your desire to bring this project to the table. This is such a game changer for all of us using peloton while indoors, but wanted to remain within Garmin ecosystem.
